Traceback (most recent call last):
  File "./hw2_tocnf.py", line 9, in <module>
    gr1 = nltk.data.load("file://"+sys.argv[1])
  File "/opt/python-3.6/lib/python3.6/site-packages/nltk/data.py", line 834, in load
    opened_resource = _open(resource_url)
  File "/opt/python-3.6/lib/python3.6/site-packages/nltk/data.py", line 955, in _open
    return find(path_, ['']).open()
  File "/opt/python-3.6/lib/python3.6/site-packages/nltk/data.py", line 673, in find
    raise LookupError(resource_not_found)
LookupError: 
**********************************************************************
  Resource [93matis.cfg[0m not found.
  Please use the NLTK Downloader to obtain the resource:

  [31m>>> import nltk
  >>> nltk.download('atis.cfg')
  [0m
  Searched in:
    - ''
**********************************************************************

